pkg/image/vulnerability_scan.go

```diff
@@ -0,0 +1,137 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package image
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"sort"
+	"strconv"
+	"strings"
+
+	buildapi "github.com/shipwright-io/build/pkg/apis/build/v1beta1"
+	"github.com/shipwright-io/build/pkg/config"
+)
+
+type TrivyVulnerability struct {
+	VulnerabilityID string `json:"vulnerabilityID,omitempty"`
+	Severity        string `json:"severity,omitempty"`
+}
+
+type TrivyResult struct {
+	Results []struct {
+		Vulnerabilities []TrivyVulnerability `json:"Vulnerabilities"`
```

Comment:
it seems we have an specific json structure here from Trivy. Does the json key need to be capitalise? e.g Vulnerabilities vs vulnerabilities?

Reply:
Done